pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -64,7 +64,7 @@
 _backends = frozenset({"cutensornet"})
 # The set of supported backends.
 
-_methods = frozenset({"mps"})
+_methods = frozenset({"mps", "tn"})
```

Comment:
exact?

Reply:
We use tn to follow the naming convention on default.tensor